security/suricata/Makefile

```diff
@@ -1,5 +1,5 @@
 # Created by: Patrick Tracanelli <eksffa@freebsdbrasil.com.br>
-# $FreeBSD$
+# $FreeBSD: head/security/suricata/Makefile 520857 2019-12-25 13:30:20Z amdmi3 $
```

Comment:
Please don't add svn macro expansions here on git

Reply:
Removed. This came over from upstream FreeBSD ports I guess.